src/domain/media/subtitle-formats/README.md

```
@@ -0,0 +1,279 @@
+  .collector(createCollector())
+  .formatter(formatToSrt)
+  .execute()
+
+console.log(pipeline.join('\n'))
+```
+
+## API Reference
+
+### Pipeline Creation
+
+#### `createPipeline(config?)`
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)